README.md

```diff
@@ -1,5 +1,8 @@
 # Foundations Course
 
+## Tentative Schedule
+[FC Schedule](https://docs.google.com/spreadsheets/d/1LsfJWwRvhkPgQox0MIep3S6L5Wi7NF0fC4IZSB0zUDQ/edit?usp=sharing)
+
```

Comment:
How is the timing for the presentation, do you think it will take 30 minutes?
Seeing the contents, I think it's doable to switch to LaTeX, could you please do that and add the changes?

Reply:
To be honest, from whatever I have read about VCS, that is most of the introduction given. Unless I need to go into Subversion or other VCS. Please do let me know if you find something missing, I will add it. The current presentation should be around 15-20 mins.

Reply:
I would add a slide about what the differences between different VCSs and why people are using git (which is a nice Segway to the next session)

Reply:
The main differences are at an abstract level(centralized vs distributed). I could add some more points to the centralized and distributed VCS slides though

Reply:
From what I have read, some of the features were better in git, for example branching. As of today, there is not much difference between the most popular VCSs and git. Please correct me if I am wrong.

Reply:
Sounds good! I think those two points should definitely be covered in the slides